src/Microsoft.ML.Transforms/Text/TextTransform.cs

```diff
@@ -116,6 +124,92 @@ public sealed class Arguments : TransformInputBase
             public TextNormKind VectorNormalizer = TextNormKind.L2;
         }
 
+        private readonly string[] _inputColumns;
+        public IReadOnlyCollection<string> InputColumns => _inputColumns.AsReadOnly();
+        public readonly string OutputColumn;
+        public readonly Language TextLanguage;
+        public readonly CaseNormalizationMode TextCase;
+        public readonly bool KeepDiacritics;
+        public readonly bool KeepPunctuations;
+        public readonly bool KeepNumbers;
+        public readonly bool OutputTokens;
+        public readonly TextNormKind VectorNormalizer;
```

Comment:
Is it absolutely necessary to make these all public?

Reply:
I would even go as far as make them mutable, but I guess this is a bit much.
But why not public? 

---
In reply to: [214522545](https://github.com/dotnet/machinelearning/pull/801#discussion_r214522545) [](ancestors = 214522545)